source/text/Format.cpp

```diff
@@ -349,12 +349,12 @@ string Format::Credits(int64_t value)
 
 // Convert the given number into abbreviated format as described in Format::Credits,
 // then attach the ' credit' or ' credits' suffix to it.
-string Format::CreditString(int64_t value)
+string Format::CreditString(int64_t value, bool abbreviated)
 {
 	if(value == 1)
 		return "1 credit";
-	else
-		return Credits(value) + " credits";
+
+	return (abbreviated ? Credits(value) : Number(value)) + " credits";
```

Comment:
We don't need the whole Format::Number here, the value is always going to be an integer.

Reply:
Are you suggesting that the namespace function FormatInteger be made a public function so that can be called instead of Format::Number?

Reply:
Nvm, I guess it doesn't like that. :pensive:

Reply:
How about now? :pleading_face: